### ISSUE TITLE
Death Stranding DLAA Mip Bias Issue

### ISSUE THREAD

**Issue by seanhouli** [emoose/DLSSTweaks#128]
When forcing DLAA (factor of 1.0), it seems there is a mip bias issue. The issue can be seen clearly with low resolution textures on Sam's model. I tried tweaking the LOD bias using NVinspector to -3, but it made no difference. I also tried other scaling factors such as 0.8, but the issue is still present.

Using latest 3.5.10 DLL override

DLSS Quality (0.6666667):
![Death Stranding Director's Cut Screenshot 2024 03 05 - 17 05 34 51](https://github.com/emoose/DLSSTweaks/assets/13198139/ad873755-abe7-4c48-86d8-6403092bd210)

DLAA (1.0):
![Death Stranding Director's Cut Screenshot 2024 03 05 - 17 06 32 78](https://github.com/emoose/DLSSTweaks/assets/13198139/131fa189-47b4-4092-afb6-8f92a12955ee)

It'd be great to get DLAA working, as DL-DSR above 4k doesn't seem to work on this game

**Comment by doktorsleepelss** [comment]
Compare them using the same preset.  DLAA is using Preset F, which is blurrier than the other presets.  That might just be what you're noticing.

**Comment by Cell-Block73** [comment]
@doktorsleepelss I agree with your assessment. Forcing anisotropic filtering solved the problem for me.
